2024-03-04 00:30:16,320 INFO    StreamThr :24133 [internal.py:wandb_internal():86] W&B internal server running at pid: 24133, started at: 2024-03-04 00:30:16.320160
2024-03-04 00:30:16,322 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status
2024-03-04 00:30:16,332 INFO    WriterThread:24133 [datastore.py:open_for_write():87] open: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/run-dvdkb3xy.wandb
2024-03-04 00:30:16,333 DEBUG   SenderThread:24133 [sender.py:send():382] send: header
2024-03-04 00:30:16,335 DEBUG   SenderThread:24133 [sender.py:send():382] send: run
2024-03-04 00:30:17,025 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: check_version
2024-03-04 00:30:17,028 INFO    SenderThread:24133 [dir_watcher.py:__init__():211] watching files in: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files
2024-03-04 00:30:17,029 INFO    SenderThread:24133 [sender.py:_start_run_threads():1136] run started: dvdkb3xy with start time 1709492416.319933
2024-03-04 00:30:17,029 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: check_version
2024-03-04 00:30:17,177 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: run_start
2024-03-04 00:30:17,181 DEBUG   HandlerThread:24133 [system_info.py:__init__():27] System info init
2024-03-04 00:30:17,181 DEBUG   HandlerThread:24133 [system_info.py:__init__():42] System info init done
2024-03-04 00:30:17,181 INFO    HandlerThread:24133 [system_monitor.py:start():194] Starting system monitor
2024-03-04 00:30:17,181 INFO    SystemMonitor:24133 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-03-04 00:30:17,182 INFO    HandlerThread:24133 [system_monitor.py:probe():214] Collecting system info
2024-03-04 00:30:17,182 INFO    SystemMonitor:24133 [interfaces.py:start():190] Started cpu monitoring
2024-03-04 00:30:17,183 INFO    SystemMonitor:24133 [interfaces.py:start():190] Started disk monitoring
2024-03-04 00:30:17,184 INFO    SystemMonitor:24133 [interfaces.py:start():190] Started gpu monitoring
2024-03-04 00:30:17,185 INFO    SystemMonitor:24133 [interfaces.py:start():190] Started memory monitoring
2024-03-04 00:30:17,187 INFO    SystemMonitor:24133 [interfaces.py:start():190] Started network monitoring
2024-03-04 00:30:17,192 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:17,192 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:17,193 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:17,194 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:17,212 DEBUG   HandlerThread:24133 [system_info.py:probe():151] Probing system
2024-03-04 00:30:17,213 DEBUG   HandlerThread:24133 [system_info.py:_probe_git():136] Probing git
2024-03-04 00:30:17,223 DEBUG   HandlerThread:24133 [system_info.py:_probe_git():144] Probing git done
2024-03-04 00:30:17,223 DEBUG   HandlerThread:24133 [system_info.py:probe():199] Probing system done
2024-03-04 00:30:17,223 DEBUG   HandlerThread:24133 [system_monitor.py:probe():223] {'os': 'Linux-5.4.0-150-generic-x86_64-with-glibc2.27', 'python': '3.11.5', 'heartbeatAt': '2024-03-03T19:00:17.212272', 'startedAt': '2024-03-03T19:00:16.312427', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '/home/sadbhawna/Desktop/cs6910_assignment1/sweep_functionality.py', 'codePathLocal': 'sweep_functionality.py', 'codePath': 'sweep_functionality.py', 'git': {'remote': 'https://github.com/AbhishekKarmakar5/cs6910_assignment1.git', 'commit': 'e66662509aa46badbf5ae8ffb71694611d656c11'}, 'email': 'YOUR_EMAIL', 'root': '/home/sadbhawna/Desktop/cs6910_assignment1', 'host': 'abhishek', 'username': 'sadbhawna', 'executable': '/home/sadbhawna/anaconda3/bin/python', 'cpu_count': 4, 'cpu_count_logical': 8, 'cpu_freq': {'current': 3.512375, 'min': 1200.0, 'max': 3800.0}, 'cpu_freq_per_core': [{'current': 3.5, 'min': 1200.0, 'max': 3800.0}, {'current': 3.549, 'min': 1200.0, 'max': 3800.0}, {'current': 3.5, 'min': 1200.0, 'max': 3800.0}, {'current': 3.5, 'min': 1200.0, 'max': 3800.0}, {'current': 3.5, 'min': 1200.0, 'max': 3800.0}, {'current': 3.55, 'min': 1200.0, 'max': 3800.0}, {'current': 3.5, 'min': 1200.0, 'max': 3800.0}, {'current': 3.5, 'min': 1200.0, 'max': 3800.0}], 'disk': {'/': {'total': 883.450553894043, 'used': 127.20074462890625}}, 'gpu': 'NVIDIA GeForce GT 710', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce GT 710', 'memory_total': 2090926080}], 'memory': {'total': 31.239574432373047}}
2024-03-04 00:30:17,223 INFO    HandlerThread:24133 [system_monitor.py:probe():224] Finished collecting system info
2024-03-04 00:30:17,223 INFO    HandlerThread:24133 [system_monitor.py:probe():227] Publishing system info
2024-03-04 00:30:17,223 DEBUG   HandlerThread:24133 [system_info.py:_save_conda():208] Saving list of conda packages installed into the current environment
2024-03-04 00:30:18,030 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_created():271] file/dir created: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/conda-environment.yaml
2024-03-04 00:30:19,195 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:19,195 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:19,195 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:19,195 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:21,196 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:21,196 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:21,196 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:21,196 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:23,196 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:23,197 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:23,197 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:23,197 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:25,197 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:25,197 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:25,197 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:25,198 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:27,198 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:27,198 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:27,198 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:27,198 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:29,199 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:29,199 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:29,199 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:29,199 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:30,450 DEBUG   HandlerThread:24133 [system_info.py:_save_conda():220] Saving conda packages done
2024-03-04 00:30:30,450 INFO    HandlerThread:24133 [system_monitor.py:probe():229] Finished publishing system info
2024-03-04 00:30:30,455 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:30:30,460 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: keepalive
2024-03-04 00:30:30,460 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:30:30,460 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: keepalive
2024-03-04 00:30:30,461 DEBUG   SenderThread:24133 [sender.py:send():382] send: files
2024-03-04 00:30:30,461 INFO    SenderThread:24133 [sender.py:_save_file():1403] saving file wandb-metadata.json with policy now
2024-03-04 00:30:30,465 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: python_packages
2024-03-04 00:30:30,465 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: stop_status
2024-03-04 00:30:30,466 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: python_packages
2024-03-04 00:30:30,466 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-04 00:30:30,468 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: stop_status
2024-03-04 00:30:30,897 DEBUG   SenderThread:24133 [sender.py:send():382] send: telemetry
2024-03-04 00:30:30,898 DEBUG   SenderThread:24133 [sender.py:send():382] send: telemetry
2024-03-04 00:30:30,898 DEBUG   SenderThread:24133 [sender.py:send():382] send: run
2024-03-04 00:30:31,032 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/config.yaml
2024-03-04 00:30:31,032 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/conda-environment.yaml
2024-03-04 00:30:31,032 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_created():271] file/dir created: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/requirements.txt
2024-03-04 00:30:31,032 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_created():271] file/dir created: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/wandb-metadata.json
2024-03-04 00:30:31,199 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:31,200 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:31,200 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:31,201 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:31,202 INFO    SenderThread:24133 [sender.py:send_run():1010] updated run: dvdkb3xy
2024-03-04 00:30:31,437 INFO    wandb-upload_0:24133 [upload_job.py:push():131] Uploaded file /tmp/tmp8kankyeswandb/ejqu16bm-wandb-metadata.json
2024-03-04 00:30:32,203 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:30:33,201 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:33,202 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:33,202 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:33,202 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:35,202 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:35,202 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:35,203 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:35,203 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:37,203 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:37,203 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:37,203 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:37,203 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:37,204 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:30:39,204 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:39,204 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:39,204 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:39,204 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:41,205 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:41,205 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:41,205 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:41,205 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:42,205 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:30:43,205 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:43,206 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:43,206 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:43,206 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:45,206 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:45,207 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:45,207 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:45,207 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:45,463 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: stop_status
2024-03-04 00:30:45,463 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: stop_status
2024-03-04 00:30:45,465 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-04 00:30:47,221 ERROR   gpu       :24133 [interfaces.py:aggregate():161] Failed to serialize metric: Not Supported
2024-03-04 00:30:47,242 ERROR   gpu       :24133 [interfaces.py:aggregate():161] Failed to serialize metric: Not Supported
2024-03-04 00:30:47,258 ERROR   gpu       :24133 [interfaces.py:aggregate():161] Failed to serialize metric: Not Supported
2024-03-04 00:30:47,258 ERROR   gpu       :24133 [interfaces.py:aggregate():161] Failed to serialize metric: division by zero
2024-03-04 00:30:47,258 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:47,258 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:47,258 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:47,259 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:47,830 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:30:49,037 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/config.yaml
2024-03-04 00:30:49,259 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:49,259 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:49,259 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:49,259 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:51,260 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:51,260 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:51,260 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:51,260 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:53,116 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:30:53,260 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:53,261 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:53,261 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:53,261 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:55,261 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:55,262 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:55,262 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:55,262 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:56,508 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: partial_history
2024-03-04 00:30:56,509 DEBUG   SenderThread:24133 [sender.py:send():382] send: history
2024-03-04 00:30:56,509 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: summary_record
2024-03-04 00:30:56,509 INFO    SenderThread:24133 [sender.py:_save_file():1403] saving file wandb-summary.json with policy end
2024-03-04 00:30:56,553 DEBUG   SenderThread:24133 [sender.py:send():382] send: config
2024-03-04 00:30:56,988 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: log_artifact
2024-03-04 00:30:56,989 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: log_artifact
2024-03-04 00:30:56,992 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: partial_history
2024-03-04 00:30:57,039 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_created():271] file/dir created: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/output.log
2024-03-04 00:30:57,039 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_created():271] file/dir created: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/media/table/confusion_matrix_table_1_52ed7f2f2ee836b83b95.table.json
2024-03-04 00:30:57,039 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_created():271] file/dir created: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/wandb-summary.json
2024-03-04 00:30:57,040 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_created():271] file/dir created: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/media/table
2024-03-04 00:30:57,040 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_created():271] file/dir created: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/media
2024-03-04 00:30:57,262 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:57,262 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:57,262 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:57,263 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:57,811 INFO    wandb-upload_0:24133 [upload_job.py:push():131] Uploaded file /tmp/tmp8kankyeswandb/pypsoqez-media/table/confusion_matrix_table_1_52ed7f2f2ee836b83b95.table.json
2024-03-04 00:30:58,124 INFO    SenderThread:24133 [sender.py:send_request_log_artifact():1470] logged artifact run-dvdkb3xy-confusion_matrix_table - {'id': 'QXJ0aWZhY3Q6NzM5NDc3MzM2', 'state': 'PENDING', 'artifactSequence': {'id': 'QXJ0aWZhY3RDb2xsZWN0aW9uOjE0NTA1MjQzNA==', 'latestArtifact': None}}
2024-03-04 00:30:58,124 DEBUG   SenderThread:24133 [sender.py:send():382] send: files
2024-03-04 00:30:58,124 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:30:58,124 INFO    SenderThread:24133 [sender.py:_save_file():1403] saving file media/table/confusion_matrix_table_1_52ed7f2f2ee836b83b95.table.json with policy now
2024-03-04 00:30:58,126 DEBUG   SenderThread:24133 [sender.py:send():382] send: history
2024-03-04 00:30:58,126 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: summary_record
2024-03-04 00:30:58,127 INFO    SenderThread:24133 [sender.py:_save_file():1403] saving file wandb-summary.json with policy end
2024-03-04 00:30:58,791 INFO    wandb-upload_1:24133 [upload_job.py:push():131] Uploaded file /tmp/tmp8kankyeswandb/it1axmm4-media/table/confusion_matrix_table_1_52ed7f2f2ee836b83b95.table.json
2024-03-04 00:30:59,040 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/output.log
2024-03-04 00:30:59,040 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/wandb-summary.json
2024-03-04 00:30:59,104 INFO    wandb-upload_0:24133 [upload_job.py:push():89] Uploaded file /home/sadbhawna/.local/share/wandb/artifacts/staging/tmps69_izfx
2024-03-04 00:30:59,263 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:59,263 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:59,263 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:30:59,263 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:00,463 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: stop_status
2024-03-04 00:31:00,463 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: stop_status
2024-03-04 00:31:00,465 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-04 00:31:01,264 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:01,264 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:01,264 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:01,264 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:03,265 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:03,265 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:03,265 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:03,265 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:03,744 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:31:05,266 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:05,266 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:05,266 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:05,266 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:07,267 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:07,267 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:07,268 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:07,268 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:08,745 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:31:09,268 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:09,268 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:09,269 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:09,269 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:11,269 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:11,270 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:11,270 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:11,270 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:13,270 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:13,270 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:13,271 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:13,271 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:13,746 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:31:15,271 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:15,271 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:15,271 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:15,271 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:15,464 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: stop_status
2024-03-04 00:31:15,465 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: stop_status
2024-03-04 00:31:15,508 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-04 00:31:17,187 DEBUG   SystemMonitor:24133 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-03-04 00:31:17,188 DEBUG   SenderThread:24133 [sender.py:send():382] send: stats
2024-03-04 00:31:17,287 ERROR   gpu       :24133 [interfaces.py:aggregate():161] Failed to serialize metric: Not Supported
2024-03-04 00:31:17,301 ERROR   gpu       :24133 [interfaces.py:aggregate():161] Failed to serialize metric: Not Supported
2024-03-04 00:31:17,331 ERROR   gpu       :24133 [interfaces.py:aggregate():161] Failed to serialize metric: Not Supported
2024-03-04 00:31:17,331 ERROR   gpu       :24133 [interfaces.py:aggregate():161] Failed to serialize metric: division by zero
2024-03-04 00:31:17,331 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:17,331 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:17,331 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:17,332 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:19,193 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:31:19,332 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:19,332 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:19,332 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:19,332 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:20,048 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/config.yaml
2024-03-04 00:31:21,333 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:21,333 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:21,333 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:21,333 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:23,334 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:23,334 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:23,334 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:23,334 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:24,493 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:31:24,803 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: partial_history
2024-03-04 00:31:24,804 DEBUG   SenderThread:24133 [sender.py:send():382] send: history
2024-03-04 00:31:24,804 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: summary_record
2024-03-04 00:31:24,804 INFO    SenderThread:24133 [sender.py:_save_file():1403] saving file wandb-summary.json with policy end
2024-03-04 00:31:24,826 DEBUG   SenderThread:24133 [sender.py:send():382] send: config
2024-03-04 00:31:25,050 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/wandb-summary.json
2024-03-04 00:31:25,244 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: log_artifact
2024-03-04 00:31:25,245 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: log_artifact
2024-03-04 00:31:25,251 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: partial_history
2024-03-04 00:31:25,334 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:25,335 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:25,335 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:25,335 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:25,861 INFO    SenderThread:24133 [sender.py:send_request_log_artifact():1470] logged artifact run-dvdkb3xy-confusion_matrix_table - {'id': 'QXJ0aWZhY3Q6NzM5NDc3OTIx', 'state': 'PENDING', 'artifactSequence': {'id': 'QXJ0aWZhY3RDb2xsZWN0aW9uOjE0NTA1MjQzNA==', 'latestArtifact': {'id': 'QXJ0aWZhY3Q6NzM5NDc3MzM2', 'versionIndex': 0}}}
2024-03-04 00:31:25,862 DEBUG   SenderThread:24133 [sender.py:send():382] send: files
2024-03-04 00:31:25,862 INFO    SenderThread:24133 [sender.py:_save_file():1403] saving file media/table/confusion_matrix_table_3_579ba1a24db539eb9a43.table.json with policy now
2024-03-04 00:31:25,865 DEBUG   SenderThread:24133 [sender.py:send():382] send: history
2024-03-04 00:31:25,865 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: summary_record
2024-03-04 00:31:25,866 INFO    SenderThread:24133 [sender.py:_save_file():1403] saving file wandb-summary.json with policy end
2024-03-04 00:31:26,050 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/wandb-summary.json
2024-03-04 00:31:26,050 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_created():271] file/dir created: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/media/table/confusion_matrix_table_3_579ba1a24db539eb9a43.table.json
2024-03-04 00:31:26,051 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/media/table
2024-03-04 00:31:26,476 INFO    wandb-upload_0:24133 [upload_job.py:push():131] Uploaded file /tmp/tmp8kankyeswandb/4yqe6gjj-media/table/confusion_matrix_table_3_579ba1a24db539eb9a43.table.json
2024-03-04 00:31:26,869 INFO    wandb-upload_1:24133 [upload_job.py:push():89] Uploaded file /home/sadbhawna/.local/share/wandb/artifacts/staging/tmpt96_bnbt
2024-03-04 00:31:27,051 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/output.log
2024-03-04 00:31:27,156 INFO    wandb-upload_0:24133 [upload_job.py:push():131] Uploaded file /tmp/tmp8kankyeswandb/qlri0qr1-media/table/confusion_matrix_table_3_579ba1a24db539eb9a43.table.json
2024-03-04 00:31:27,335 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:27,335 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:27,336 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:27,336 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:29,336 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:29,336 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:29,336 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:29,336 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:29,867 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:31:30,465 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: stop_status
2024-03-04 00:31:30,466 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: stop_status
2024-03-04 00:31:30,469 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-04 00:31:31,337 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:31,337 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:31,337 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:31,337 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:33,338 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:33,338 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:33,338 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:33,338 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:35,338 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:35,339 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:35,339 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:35,339 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:35,787 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:31:37,339 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:37,340 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:37,340 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:37,340 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:39,340 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:39,340 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:39,341 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:39,341 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:40,788 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:31:41,341 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:41,341 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:41,341 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:41,341 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:43,342 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:43,342 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:43,342 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:43,342 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:45,343 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:45,343 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:45,343 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:45,343 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:45,466 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: stop_status
2024-03-04 00:31:45,466 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: stop_status
2024-03-04 00:31:45,508 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-04 00:31:45,827 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:31:46,252 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: partial_history
2024-03-04 00:31:46,253 DEBUG   SenderThread:24133 [sender.py:send():382] send: history
2024-03-04 00:31:46,253 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: summary_record
2024-03-04 00:31:46,253 INFO    SenderThread:24133 [sender.py:_save_file():1403] saving file wandb-summary.json with policy end
2024-03-04 00:31:46,275 DEBUG   SenderThread:24133 [sender.py:send():382] send: config
2024-03-04 00:31:46,643 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: log_artifact
2024-03-04 00:31:46,643 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: log_artifact
2024-03-04 00:31:46,657 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: partial_history
2024-03-04 00:31:47,059 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/wandb-summary.json
2024-03-04 00:31:47,059 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_created():271] file/dir created: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/media/table/confusion_matrix_table_5_d9f7053150ccbacb87ed.table.json
2024-03-04 00:31:47,059 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/media/table
2024-03-04 00:31:47,226 INFO    SenderThread:24133 [sender.py:send_request_log_artifact():1470] logged artifact run-dvdkb3xy-confusion_matrix_table - {'id': 'QXJ0aWZhY3Q6NzM5NDc4MzQ4', 'state': 'PENDING', 'artifactSequence': {'id': 'QXJ0aWZhY3RDb2xsZWN0aW9uOjE0NTA1MjQzNA==', 'latestArtifact': {'id': 'QXJ0aWZhY3Q6NzM5NDc3OTIx', 'versionIndex': 1}}}
2024-03-04 00:31:47,226 DEBUG   SenderThread:24133 [sender.py:send():382] send: files
2024-03-04 00:31:47,226 INFO    SenderThread:24133 [sender.py:_save_file():1403] saving file media/table/confusion_matrix_table_5_d9f7053150ccbacb87ed.table.json with policy now
2024-03-04 00:31:47,226 DEBUG   SenderThread:24133 [sender.py:send():382] send: history
2024-03-04 00:31:47,227 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: summary_record
2024-03-04 00:31:47,229 INFO    SenderThread:24133 [sender.py:_save_file():1403] saving file wandb-summary.json with policy end
2024-03-04 00:31:47,229 DEBUG   SenderThread:24133 [sender.py:send():382] send: stats
2024-03-04 00:31:47,358 ERROR   gpu       :24133 [interfaces.py:aggregate():161] Failed to serialize metric: Not Supported
2024-03-04 00:31:47,373 ERROR   gpu       :24133 [interfaces.py:aggregate():161] Failed to serialize metric: Not Supported
2024-03-04 00:31:47,388 ERROR   gpu       :24133 [interfaces.py:aggregate():161] Failed to serialize metric: Not Supported
2024-03-04 00:31:47,389 ERROR   gpu       :24133 [interfaces.py:aggregate():161] Failed to serialize metric: division by zero
2024-03-04 00:31:47,389 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:47,389 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:47,389 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:47,389 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:47,647 INFO    wandb-upload_1:24133 [upload_job.py:push():131] Uploaded file /tmp/tmp8kankyeswandb/l3y4hpzw-media/table/confusion_matrix_table_5_d9f7053150ccbacb87ed.table.json
2024-03-04 00:31:48,060 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/wandb-summary.json
2024-03-04 00:31:48,296 INFO    wandb-upload_0:24133 [upload_job.py:push():89] Uploaded file /home/sadbhawna/.local/share/wandb/artifacts/staging/tmpp9vln9si
2024-03-04 00:31:48,316 INFO    wandb-upload_1:24133 [upload_job.py:push():131] Uploaded file /tmp/tmp8kankyeswandb/6w8oyukq-media/table/confusion_matrix_table_5_d9f7053150ccbacb87ed.table.json
2024-03-04 00:31:49,061 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/output.log
2024-03-04 00:31:49,390 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:49,390 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:49,390 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:49,390 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:51,234 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:31:51,390 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:51,390 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:51,391 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:51,391 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:52,063 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/config.yaml
2024-03-04 00:31:53,391 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:53,391 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:53,392 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:53,392 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:55,392 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:55,392 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:55,393 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:55,393 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:56,517 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:31:57,393 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:57,393 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:57,393 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:57,394 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:59,394 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:59,394 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:59,394 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:31:59,394 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:00,466 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: stop_status
2024-03-04 00:32:00,467 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: stop_status
2024-03-04 00:32:00,469 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-04 00:32:01,395 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:01,395 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:01,395 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:01,395 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:01,789 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:32:03,396 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:03,396 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:03,396 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:03,396 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:05,397 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:05,397 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:05,397 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:05,397 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:06,790 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:32:07,320 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: partial_history
2024-03-04 00:32:07,320 DEBUG   SenderThread:24133 [sender.py:send():382] send: history
2024-03-04 00:32:07,321 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: summary_record
2024-03-04 00:32:07,321 INFO    SenderThread:24133 [sender.py:_save_file():1403] saving file wandb-summary.json with policy end
2024-03-04 00:32:07,346 DEBUG   SenderThread:24133 [sender.py:send():382] send: config
2024-03-04 00:32:07,397 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:07,398 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:07,398 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:07,398 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:07,784 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: log_artifact
2024-03-04 00:32:07,785 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: log_artifact
2024-03-04 00:32:07,789 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: partial_history
2024-03-04 00:32:08,070 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/wandb-summary.json
2024-03-04 00:32:08,070 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_created():271] file/dir created: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/media/table/confusion_matrix_table_7_d240edccde282b87d1d2.table.json
2024-03-04 00:32:08,070 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/media/table
2024-03-04 00:32:08,352 INFO    SenderThread:24133 [sender.py:send_request_log_artifact():1470] logged artifact run-dvdkb3xy-confusion_matrix_table - {'id': 'QXJ0aWZhY3Q6NzM5NDc4ODE3', 'state': 'PENDING', 'artifactSequence': {'id': 'QXJ0aWZhY3RDb2xsZWN0aW9uOjE0NTA1MjQzNA==', 'latestArtifact': {'id': 'QXJ0aWZhY3Q6NzM5NDc4MzQ4', 'versionIndex': 2}}}
2024-03-04 00:32:08,353 DEBUG   SenderThread:24133 [sender.py:send():382] send: files
2024-03-04 00:32:08,353 INFO    SenderThread:24133 [sender.py:_save_file():1403] saving file media/table/confusion_matrix_table_7_d240edccde282b87d1d2.table.json with policy now
2024-03-04 00:32:08,354 DEBUG   SenderThread:24133 [sender.py:send():382] send: history
2024-03-04 00:32:08,354 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: summary_record
2024-03-04 00:32:08,354 INFO    SenderThread:24133 [sender.py:_save_file():1403] saving file wandb-summary.json with policy end
2024-03-04 00:32:08,850 INFO    wandb-upload_0:24133 [upload_job.py:push():131] Uploaded file /tmp/tmp8kankyeswandb/ez38fil3-media/table/confusion_matrix_table_7_d240edccde282b87d1d2.table.json
2024-03-04 00:32:09,072 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/output.log
2024-03-04 00:32:09,072 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/wandb-summary.json
2024-03-04 00:32:09,398 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:09,398 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:09,398 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:09,399 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:09,403 INFO    wandb-upload_1:24133 [upload_job.py:push():89] Uploaded file /home/sadbhawna/.local/share/wandb/artifacts/staging/tmp9ub_gshe
2024-03-04 00:32:09,684 INFO    wandb-upload_0:24133 [upload_job.py:push():131] Uploaded file /tmp/tmp8kankyeswandb/uga5gy7y-media/table/confusion_matrix_table_7_d240edccde282b87d1d2.table.json
2024-03-04 00:32:11,399 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:11,399 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:11,399 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:11,399 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:12,355 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:32:13,400 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:13,400 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:13,400 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:13,400 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:15,401 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:15,401 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:15,401 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:15,401 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:15,466 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: stop_status
2024-03-04 00:32:15,467 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: stop_status
2024-03-04 00:32:15,470 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-04 00:32:17,189 DEBUG   SenderThread:24133 [sender.py:send():382] send: stats
2024-03-04 00:32:17,416 ERROR   gpu       :24133 [interfaces.py:aggregate():161] Failed to serialize metric: Not Supported
2024-03-04 00:32:17,430 ERROR   gpu       :24133 [interfaces.py:aggregate():161] Failed to serialize metric: Not Supported
2024-03-04 00:32:17,449 ERROR   gpu       :24133 [interfaces.py:aggregate():161] Failed to serialize metric: Not Supported
2024-03-04 00:32:17,449 ERROR   gpu       :24133 [interfaces.py:aggregate():161] Failed to serialize metric: division by zero
2024-03-04 00:32:17,450 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:17,450 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:17,450 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:17,450 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:18,190 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:32:19,450 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:19,451 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:19,451 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:19,451 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:21,451 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:21,451 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:21,451 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:21,451 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:23,195 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:32:23,452 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:23,452 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:23,452 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:23,452 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:24,078 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/config.yaml
2024-03-04 00:32:25,453 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:25,453 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:25,453 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:25,453 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:27,453 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:27,454 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:27,454 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:27,454 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:28,491 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:32:29,454 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:29,454 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:29,455 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:29,455 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:30,466 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: stop_status
2024-03-04 00:32:30,467 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: stop_status
2024-03-04 00:32:30,508 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-04 00:32:31,455 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:31,455 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:31,455 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:31,456 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:33,456 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:33,456 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:33,456 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:33,456 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:33,823 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:32:35,457 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:35,457 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:35,457 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:35,457 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:37,409 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: partial_history
2024-03-04 00:32:37,410 DEBUG   SenderThread:24133 [sender.py:send():382] send: history
2024-03-04 00:32:37,410 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: summary_record
2024-03-04 00:32:37,412 INFO    SenderThread:24133 [sender.py:_save_file():1403] saving file wandb-summary.json with policy end
2024-03-04 00:32:37,447 DEBUG   SenderThread:24133 [sender.py:send():382] send: config
2024-03-04 00:32:37,458 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:37,458 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:37,458 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:37,458 ERROR   gpu       :24133 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-04 00:32:37,876 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: log_artifact
2024-03-04 00:32:37,877 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: log_artifact
2024-03-04 00:32:37,885 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: partial_history
2024-03-04 00:32:38,084 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/wandb-summary.json
2024-03-04 00:32:38,085 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_created():271] file/dir created: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/media/table/confusion_matrix_table_9_48fa334e8b64e1857904.table.json
2024-03-04 00:32:38,085 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/media/table
2024-03-04 00:32:38,479 INFO    SenderThread:24133 [sender.py:send_request_log_artifact():1470] logged artifact run-dvdkb3xy-confusion_matrix_table - {'id': 'QXJ0aWZhY3Q6NzM5NDc5NDUy', 'state': 'PENDING', 'artifactSequence': {'id': 'QXJ0aWZhY3RDb2xsZWN0aW9uOjE0NTA1MjQzNA==', 'latestArtifact': {'id': 'QXJ0aWZhY3Q6NzM5NDc4ODE3', 'versionIndex': 3}}}
2024-03-04 00:32:38,480 DEBUG   SenderThread:24133 [sender.py:send():382] send: files
2024-03-04 00:32:38,480 INFO    SenderThread:24133 [sender.py:_save_file():1403] saving file media/table/confusion_matrix_table_9_48fa334e8b64e1857904.table.json with policy now
2024-03-04 00:32:38,481 DEBUG   SenderThread:24133 [sender.py:send():382] send: history
2024-03-04 00:32:38,481 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: summary_record
2024-03-04 00:32:38,481 INFO    SenderThread:24133 [sender.py:_save_file():1403] saving file wandb-summary.json with policy end
2024-03-04 00:32:38,481 DEBUG   SenderThread:24133 [sender.py:send():382] send: telemetry
2024-03-04 00:32:38,482 DEBUG   SenderThread:24133 [sender.py:send():382] send: telemetry
2024-03-04 00:32:38,482 DEBUG   SenderThread:24133 [sender.py:send():382] send: exit
2024-03-04 00:32:38,482 INFO    SenderThread:24133 [sender.py:send_exit():589] handling exit code: 0
2024-03-04 00:32:38,482 INFO    SenderThread:24133 [sender.py:send_exit():591] handling runtime: 140
2024-03-04 00:32:38,482 INFO    SenderThread:24133 [sender.py:_save_file():1403] saving file wandb-summary.json with policy end
2024-03-04 00:32:38,482 INFO    SenderThread:24133 [sender.py:send_exit():597] send defer
2024-03-04 00:32:38,483 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: defer
2024-03-04 00:32:38,483 INFO    HandlerThread:24133 [handler.py:handle_request_defer():172] handle defer: 0
2024-03-04 00:32:38,483 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: defer
2024-03-04 00:32:38,483 INFO    SenderThread:24133 [sender.py:send_request_defer():613] handle sender defer: 0
2024-03-04 00:32:38,483 INFO    SenderThread:24133 [sender.py:transition_state():617] send defer: 1
2024-03-04 00:32:38,483 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: defer
2024-03-04 00:32:38,483 INFO    HandlerThread:24133 [handler.py:handle_request_defer():172] handle defer: 1
2024-03-04 00:32:38,483 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: defer
2024-03-04 00:32:38,483 INFO    SenderThread:24133 [sender.py:send_request_defer():613] handle sender defer: 1
2024-03-04 00:32:38,483 INFO    SenderThread:24133 [sender.py:transition_state():617] send defer: 2
2024-03-04 00:32:38,484 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: defer
2024-03-04 00:32:38,484 INFO    HandlerThread:24133 [handler.py:handle_request_defer():172] handle defer: 2
2024-03-04 00:32:38,484 INFO    HandlerThread:24133 [system_monitor.py:finish():203] Stopping system monitor
2024-03-04 00:32:38,484 DEBUG   SystemMonitor:24133 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-03-04 00:32:38,484 DEBUG   SystemMonitor:24133 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-03-04 00:32:38,485 INFO    HandlerThread:24133 [interfaces.py:finish():202] Joined cpu monitor
2024-03-04 00:32:38,485 INFO    HandlerThread:24133 [interfaces.py:finish():202] Joined disk monitor
2024-03-04 00:32:38,501 ERROR   gpu       :24133 [interfaces.py:aggregate():161] Failed to serialize metric: Not Supported
2024-03-04 00:32:38,518 ERROR   gpu       :24133 [interfaces.py:aggregate():161] Failed to serialize metric: Not Supported
2024-03-04 00:32:38,539 ERROR   gpu       :24133 [interfaces.py:aggregate():161] Failed to serialize metric: Not Supported
2024-03-04 00:32:38,540 ERROR   gpu       :24133 [interfaces.py:aggregate():161] Failed to serialize metric: division by zero
2024-03-04 00:32:38,540 INFO    HandlerThread:24133 [interfaces.py:finish():202] Joined gpu monitor
2024-03-04 00:32:38,540 INFO    HandlerThread:24133 [interfaces.py:finish():202] Joined memory monitor
2024-03-04 00:32:38,540 INFO    HandlerThread:24133 [interfaces.py:finish():202] Joined network monitor
2024-03-04 00:32:38,541 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: defer
2024-03-04 00:32:38,541 INFO    SenderThread:24133 [sender.py:send_request_defer():613] handle sender defer: 2
2024-03-04 00:32:38,541 INFO    SenderThread:24133 [sender.py:transition_state():617] send defer: 3
2024-03-04 00:32:38,541 DEBUG   SenderThread:24133 [sender.py:send():382] send: stats
2024-03-04 00:32:38,541 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: defer
2024-03-04 00:32:38,541 INFO    HandlerThread:24133 [handler.py:handle_request_defer():172] handle defer: 3
2024-03-04 00:32:38,542 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: defer
2024-03-04 00:32:38,542 INFO    SenderThread:24133 [sender.py:send_request_defer():613] handle sender defer: 3
2024-03-04 00:32:38,542 INFO    SenderThread:24133 [sender.py:transition_state():617] send defer: 4
2024-03-04 00:32:38,542 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: defer
2024-03-04 00:32:38,542 INFO    HandlerThread:24133 [handler.py:handle_request_defer():172] handle defer: 4
2024-03-04 00:32:38,542 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: defer
2024-03-04 00:32:38,542 INFO    SenderThread:24133 [sender.py:send_request_defer():613] handle sender defer: 4
2024-03-04 00:32:38,542 INFO    SenderThread:24133 [sender.py:transition_state():617] send defer: 5
2024-03-04 00:32:38,542 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: defer
2024-03-04 00:32:38,542 INFO    HandlerThread:24133 [handler.py:handle_request_defer():172] handle defer: 5
2024-03-04 00:32:38,543 DEBUG   SenderThread:24133 [sender.py:send():382] send: summary
2024-03-04 00:32:38,543 INFO    SenderThread:24133 [sender.py:_save_file():1403] saving file wandb-summary.json with policy end
2024-03-04 00:32:38,543 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: defer
2024-03-04 00:32:38,543 INFO    SenderThread:24133 [sender.py:send_request_defer():613] handle sender defer: 5
2024-03-04 00:32:38,543 INFO    SenderThread:24133 [sender.py:transition_state():617] send defer: 6
2024-03-04 00:32:38,543 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: defer
2024-03-04 00:32:38,544 INFO    HandlerThread:24133 [handler.py:handle_request_defer():172] handle defer: 6
2024-03-04 00:32:38,544 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: defer
2024-03-04 00:32:38,544 INFO    SenderThread:24133 [sender.py:send_request_defer():613] handle sender defer: 6
2024-03-04 00:32:38,548 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: status_report
2024-03-04 00:32:38,675 INFO    wandb-upload_1:24133 [upload_job.py:push():131] Uploaded file /tmp/tmp8kankyeswandb/qsvxs2ip-media/table/confusion_matrix_table_9_48fa334e8b64e1857904.table.json
2024-03-04 00:32:38,836 INFO    SenderThread:24133 [sender.py:transition_state():617] send defer: 7
2024-03-04 00:32:38,836 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: defer
2024-03-04 00:32:38,836 INFO    HandlerThread:24133 [handler.py:handle_request_defer():172] handle defer: 7
2024-03-04 00:32:38,836 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: defer
2024-03-04 00:32:38,836 INFO    SenderThread:24133 [sender.py:send_request_defer():613] handle sender defer: 7
2024-03-04 00:32:38,890 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: poll_exit
2024-03-04 00:32:39,086 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/output.log
2024-03-04 00:32:39,086 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/wandb-summary.json
2024-03-04 00:32:39,086 INFO    Thread-13 :24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/config.yaml
2024-03-04 00:32:39,439 INFO    wandb-upload_1:24133 [upload_job.py:push():131] Uploaded file /tmp/tmp8kankyeswandb/oh2146gq-media/table/confusion_matrix_table_9_48fa334e8b64e1857904.table.json
2024-03-04 00:32:39,482 INFO    wandb-upload_0:24133 [upload_job.py:push():89] Uploaded file /home/sadbhawna/.local/share/wandb/artifacts/staging/tmpfk6mdqhx
2024-03-04 00:32:40,514 INFO    SenderThread:24133 [sender.py:transition_state():617] send defer: 8
2024-03-04 00:32:40,515 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: poll_exit
2024-03-04 00:32:40,515 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: defer
2024-03-04 00:32:40,515 INFO    HandlerThread:24133 [handler.py:handle_request_defer():172] handle defer: 8
2024-03-04 00:32:40,515 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: defer
2024-03-04 00:32:40,515 INFO    SenderThread:24133 [sender.py:send_request_defer():613] handle sender defer: 8
2024-03-04 00:32:40,515 INFO    SenderThread:24133 [sender.py:transition_state():617] send defer: 9
2024-03-04 00:32:40,516 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: defer
2024-03-04 00:32:40,516 INFO    HandlerThread:24133 [handler.py:handle_request_defer():172] handle defer: 9
2024-03-04 00:32:40,516 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: defer
2024-03-04 00:32:40,516 INFO    SenderThread:24133 [sender.py:send_request_defer():613] handle sender defer: 9
2024-03-04 00:32:40,516 INFO    SenderThread:24133 [dir_watcher.py:finish():358] shutting down directory watcher
2024-03-04 00:32:40,891 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: poll_exit
2024-03-04 00:32:41,087 INFO    SenderThread:24133 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/output.log
2024-03-04 00:32:41,088 INFO    SenderThread:24133 [dir_watcher.py:finish():388] scan: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files
2024-03-04 00:32:41,088 INFO    SenderThread:24133 [dir_watcher.py:finish():402] scan save: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/wandb-metadata.json wandb-metadata.json
2024-03-04 00:32:41,088 INFO    SenderThread:24133 [dir_watcher.py:finish():402] scan save: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/requirements.txt requirements.txt
2024-03-04 00:32:41,088 INFO    SenderThread:24133 [dir_watcher.py:finish():402] scan save: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/conda-environment.yaml conda-environment.yaml
2024-03-04 00:32:41,090 INFO    SenderThread:24133 [dir_watcher.py:finish():402] scan save: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/config.yaml config.yaml
2024-03-04 00:32:41,093 INFO    SenderThread:24133 [dir_watcher.py:finish():402] scan save: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/output.log output.log
2024-03-04 00:32:41,095 INFO    SenderThread:24133 [dir_watcher.py:finish():402] scan save: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/wandb-summary.json wandb-summary.json
2024-03-04 00:32:41,108 INFO    SenderThread:24133 [dir_watcher.py:finish():402] scan save: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/media/table/confusion_matrix_table_5_d9f7053150ccbacb87ed.table.json media/table/confusion_matrix_table_5_d9f7053150ccbacb87ed.table.json
2024-03-04 00:32:41,108 INFO    SenderThread:24133 [dir_watcher.py:finish():402] scan save: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/media/table/confusion_matrix_table_1_52ed7f2f2ee836b83b95.table.json media/table/confusion_matrix_table_1_52ed7f2f2ee836b83b95.table.json
2024-03-04 00:32:41,108 INFO    SenderThread:24133 [dir_watcher.py:finish():402] scan save: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/media/table/confusion_matrix_table_7_d240edccde282b87d1d2.table.json media/table/confusion_matrix_table_7_d240edccde282b87d1d2.table.json
2024-03-04 00:32:41,109 INFO    SenderThread:24133 [dir_watcher.py:finish():402] scan save: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/media/table/confusion_matrix_table_3_579ba1a24db539eb9a43.table.json media/table/confusion_matrix_table_3_579ba1a24db539eb9a43.table.json
2024-03-04 00:32:41,109 INFO    SenderThread:24133 [dir_watcher.py:finish():402] scan save: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/media/table/confusion_matrix_table_9_48fa334e8b64e1857904.table.json media/table/confusion_matrix_table_9_48fa334e8b64e1857904.table.json
2024-03-04 00:32:41,109 INFO    SenderThread:24133 [sender.py:transition_state():617] send defer: 10
2024-03-04 00:32:41,109 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: poll_exit
2024-03-04 00:32:41,109 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: defer
2024-03-04 00:32:41,112 INFO    HandlerThread:24133 [handler.py:handle_request_defer():172] handle defer: 10
2024-03-04 00:32:41,116 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: defer
2024-03-04 00:32:41,116 INFO    SenderThread:24133 [sender.py:send_request_defer():613] handle sender defer: 10
2024-03-04 00:32:41,116 INFO    SenderThread:24133 [file_pusher.py:finish():172] shutting down file pusher
2024-03-04 00:32:41,708 INFO    wandb-upload_0:24133 [upload_job.py:push():131] Uploaded file /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/conda-environment.yaml
2024-03-04 00:32:41,801 INFO    wandb-upload_2:24133 [upload_job.py:push():131] Uploaded file /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/config.yaml
2024-03-04 00:32:41,805 INFO    wandb-upload_1:24133 [upload_job.py:push():131] Uploaded file /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/requirements.txt
2024-03-04 00:32:41,891 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: poll_exit
2024-03-04 00:32:41,892 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: poll_exit
2024-03-04 00:32:41,899 INFO    wandb-upload_4:24133 [upload_job.py:push():131] Uploaded file /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/wandb-summary.json
2024-03-04 00:32:41,937 INFO    wandb-upload_3:24133 [upload_job.py:push():131] Uploaded file /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/files/output.log
2024-03-04 00:32:42,137 INFO    Thread-12 (_thread_body):24133 [sender.py:transition_state():617] send defer: 11
2024-03-04 00:32:42,138 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: defer
2024-03-04 00:32:42,138 INFO    HandlerThread:24133 [handler.py:handle_request_defer():172] handle defer: 11
2024-03-04 00:32:42,138 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: defer
2024-03-04 00:32:42,138 INFO    SenderThread:24133 [sender.py:send_request_defer():613] handle sender defer: 11
2024-03-04 00:32:42,138 INFO    SenderThread:24133 [file_pusher.py:join():178] waiting for file pusher
2024-03-04 00:32:42,139 INFO    SenderThread:24133 [sender.py:transition_state():617] send defer: 12
2024-03-04 00:32:42,139 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: defer
2024-03-04 00:32:42,140 INFO    HandlerThread:24133 [handler.py:handle_request_defer():172] handle defer: 12
2024-03-04 00:32:42,140 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: defer
2024-03-04 00:32:42,140 INFO    SenderThread:24133 [sender.py:send_request_defer():613] handle sender defer: 12
2024-03-04 00:32:42,140 INFO    SenderThread:24133 [file_stream.py:finish():595] file stream finish called
2024-03-04 00:32:42,892 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: poll_exit
2024-03-04 00:32:43,044 INFO    SenderThread:24133 [file_stream.py:finish():599] file stream finish is done
2024-03-04 00:32:43,044 INFO    SenderThread:24133 [sender.py:transition_state():617] send defer: 13
2024-03-04 00:32:43,045 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: poll_exit
2024-03-04 00:32:43,045 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: defer
2024-03-04 00:32:43,045 INFO    HandlerThread:24133 [handler.py:handle_request_defer():172] handle defer: 13
2024-03-04 00:32:43,045 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: defer
2024-03-04 00:32:43,045 INFO    SenderThread:24133 [sender.py:send_request_defer():613] handle sender defer: 13
2024-03-04 00:32:43,045 INFO    SenderThread:24133 [sender.py:transition_state():617] send defer: 14
2024-03-04 00:32:43,046 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: defer
2024-03-04 00:32:43,046 DEBUG   SenderThread:24133 [sender.py:send():382] send: final
2024-03-04 00:32:43,046 INFO    HandlerThread:24133 [handler.py:handle_request_defer():172] handle defer: 14
2024-03-04 00:32:43,046 DEBUG   SenderThread:24133 [sender.py:send():382] send: footer
2024-03-04 00:32:43,046 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: defer
2024-03-04 00:32:43,046 INFO    SenderThread:24133 [sender.py:send_request_defer():613] handle sender defer: 14
2024-03-04 00:32:43,047 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: poll_exit
2024-03-04 00:32:43,047 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: poll_exit
2024-03-04 00:32:43,048 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-04 00:32:43,048 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: server_info
2024-03-04 00:32:43,049 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: get_summary
2024-03-04 00:32:43,049 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: sampled_history
2024-03-04 00:32:43,049 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: job_info
2024-03-04 00:32:43,050 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: server_info
2024-03-04 00:32:43,306 DEBUG   SenderThread:24133 [sender.py:send_request():409] send_request: job_info
2024-03-04 00:32:43,307 DEBUG   HandlerThread:24133 [handler.py:handle_request():146] handle_request: shutdown
2024-03-04 00:32:43,307 INFO    HandlerThread:24133 [handler.py:finish():869] shutting down handler
2024-03-04 00:32:44,050 INFO    WriterThread:24133 [datastore.py:close():296] close: /home/sadbhawna/Desktop/cs6910_assignment1/wandb/run-20240304_003016-dvdkb3xy/run-dvdkb3xy.wandb
2024-03-04 00:32:44,306 INFO    SenderThread:24133 [sender.py:finish():1572] shutting down sender
2024-03-04 00:32:44,306 INFO    SenderThread:24133 [file_pusher.py:finish():172] shutting down file pusher
2024-03-04 00:32:44,306 INFO    SenderThread:24133 [file_pusher.py:join():178] waiting for file pusher
